nss-day-cohort-39/nutshell-zaffre-zorillas
# Cleaned up code and fixed Event errors

### Changes
- Added second parameter to Event function to render CSS styling on "nextEvent" class
- Added userId key to newEvent object to collect session storage user for event filtering
- Fixed conditions for data capture in new event fields (all now required before event will save)
- Added pseudo code to all modules and tidied up code indention and brackets

### Testing
- Make sure you can add and save a new event and that it renders immediately on the page
- Confirm that the nearest event is highlighted in gray and has bolder, bigger text
- Make sure the delete event button works
